performance_regression_test.py

```diff
@@ -146,9 +146,16 @@ def generate_stats_json(self, results, cmds=[]):
         metrics['test_details']['sct_git_commit'] = \
             subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
 
-        if os.environ.get('JOB_NAME'):
-            metrics['test_details']['job_name'] = os.environ.get('JOB_NAME', '')
+        job_name = os.environ.get('JOB_NAME', 'local_run')
+        metrics['test_details']['job_name'] = job_name
+        if os.environ.get('BUILD_URL'):
             metrics['test_details']['job_url'] = os.environ.get('BUILD_URL', '')
+        if self.monitors:
+            zip_file = os.path.join(self.logdir, os.path.basename(os.path.normpath(self.job.logdir)))
```

Comment:
The way to get the url_s3 from self.logdir is specific, and it's used in many place of this patch, can we add a helper function to do this?